ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/Common/SPFocusableTextView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SPFocusableTextView.swift
+//  ConsentViewController-tvOS
+//
+//  Created by Andre Herculano on 17.03.22.
+//
+
+import UIKit
+
+class SPFocusableTextView: UITextView, UITextViewDelegate {
+    override var canBecomeFocused: Bool { isContentBig }
+    var isContentBig: Bool { self.contentSize.height > self.frame.size.height }
+    public var canFocusCategoryTableView: Bool = true
+
+    open override func didUpdateFocus(in context: UIFocusUpdateContext, with coordinator: UIFocusAnimationCoordinator) {
+        if isFocused {
+            backgroundColor = .lightGray.withAlphaComponent(0.5)
+            flashScrollIndicators()
+            if isContentBig {
+                canFocusCategoryTableView = false
```

Comment:
@Nevazhnovu having a property called `canFocusCategoryTableView` suggests `SPFocusableTextView` knows about something external to its own responsibilities.

I suggest renaming the field to something like `contentFitsContainer` . The same applies to the method `checkIfCategoryTableViewFocusable`. It should be renamed to something like `updateContentFitsContainer`.

Reply:
Done